MSAL/src/MSALPublicClientApplication.m

```diff
@@ -541,24 +577,25 @@ - (BOOL)removeUser:(MSALUser *)user
         return YES;
     }
     
-    return [_tokenCache deleteAllTokensForUser:user clientId:self.clientId context:nil error:error];
+    __auto_type tokens = [self.tokenCache allTokensForAccount:user.account context:nil error:error];
```

Comment:
this doesn't need to be two step, we can optimize by doing deletion in one step.

Reply:
done